pyproject.toml

```
@@ -150,6 +150,18 @@ version = { attr = "integreat_compass.__version__" }
 [tool.setuptools.packages.find]
 include = ["integreat_compass*"]
 
+[tool.bumpver]
+current_version = "2023.0.0"
```

```suggestion
current_version = "2023.3.0"
```
0 is not a valid month. Please keep the current version and only bump it when there is a real release. Merging this PR does not create a new release, so it should also not change the version.